templates/workspace_services/guacamole/user_resources/guacamole-azure-export-reviewvm/porter.yaml

```diff
@@ -0,0 +1,237 @@
+  - name: os_image
+    type: string
+    default: "Windows 10"
+  - name: vm_size
+    type: string
+    default: "2 CPU | 8GB RAM"
+
+outputs:
+  - name: ip
+    type: string
+    applyTo:
+      - install
+  - name: hostname
+    type: string
+    applyTo:
+      - install
+  - name: connection_uri
+    type: string
+    applyTo:
+      - install
+  - name: azure_resource_id
+    type: string
+    applyTo:
+      - install
+      - start
+      - stop
+      - reset_password
+
+mixins:
+  - exec
+  - terraform:
+      clientVersion: 1.2.8
+  - az
+
+install:
+  - terraform:
+      description: "Deploy Guacamole User Resource Service (Reviewer VM)"
+      vars:
+        workspace_id: "{{ bundle.parameters.workspace_id }}"
+        tre_id: "{{ bundle.parameters.tre_id }}"
+        parent_service_id: "{{ bundle.parameters.parent_service_id }}"
+        tre_resource_id: "{{ bundle.parameters.id }}"
+        image: "{{ bundle.parameters.os_image }}"
+        vm_size: "{{ bundle.parameters.vm_size }}"
+        shared_storage_access: "false"
+        shared_storage_name: "unused"
+      backendConfig:
+        resource_group_name: "{{ bundle.parameters.tfstate_resource_group_name }}"
+        storage_account_name: "{{ bundle.parameters.tfstate_storage_account_name }}"
+        container_name: "{{ bundle.parameters.tfstate_container_name }}"
+        key: "{{ bundle.parameters.id }}"
+      outputs:
+        - name: ip
+        - name: hostname
+        - name: connection_uri
+        - name: azure_resource_id
+
+upgrade:
+  - terraform:
+      description: "Update Guacamole User Resource Service (Reviewer VM)"
+      vars:
+        workspace_id: "{{ bundle.parameters.workspace_id }}"
+        tre_id: "{{ bundle.parameters.tre_id }}"
+        parent_service_id: "{{ bundle.parameters.parent_service_id }}"
+        tre_resource_id: "{{ bundle.parameters.id }}"
+        image: "{{ bundle.parameters.os_image }}"
+        vm_size: "{{ bundle.parameters.vm_size }}"
+        shared_storage_access: "false"
+        shared_storage_name: "unused"
+      backendConfig:
+        resource_group_name: "{{ bundle.parameters.tfstate_resource_group_name }}"
+        storage_account_name: "{{ bundle.parameters.tfstate_storage_account_name }}"
+        container_name: "{{ bundle.parameters.tfstate_container_name }}"
+        key: "{{ bundle.parameters.id }}"
+      outputs:
+        - name: ip
+        - name: hostname
+        - name: connection_uri
+        - name: azure_resource_id
+
+uninstall:
+  - exec:
+      description: "Delete the Extensions from the Terraform state manually"
+      command: ./delete_vm_extensions.sh
+      arguments:
+        - "{{ bundle.parameters.tfstate_resource_group_name }}"
+        - "{{ bundle.parameters.tfstate_storage_account_name }}"
+        - "{{ bundle.parameters.tfstate_container_name }}"
+        - "{{ bundle.parameters.id }}"
+
+  - terraform:
+      description: "Delete the Guacamole User Resource Service"
```

Comment:
you've added `(Reviewer VM)` to the other descriptions
did you want to add it here too?

Reply:
Good catch